bootstrap.sh

```diff
@@ -46,6 +46,3 @@ cp -r .githooks/* .git/hooks/
 
 # Make the hooks are executable
 chmod +x .git/hooks/*
-
-# Run and update content blocker
-./content_blocker_update.sh
```

Comment:
cc @clarmso Just a quick heads up that we are removing this content blocker update script from the bootstrap.sh. 

I'm not aware of any issues this should cause on Bitrise but wanted to make sure you were looped in. Thanks

Reply:
I just noticed that this script is also responsible for running the code that generates our `MainFrameAtDocument*.js` files. So I'm fixing this right now, should have an update pushed soon.

Reply:
(Update: the above issue should be fixed now.)